.github/workflows/publish-streamer.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish new version of Firecrest Streamer
+
+on:
+  push:
+    branches: [ "master" ]
+    paths:
+          - "src/firecrest-streamer/pyproject.toml"
+          - "src/firecrest-streamer/src/**"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build toml requests
+
+      - name: Check PyPI for latest version
+        id: check_version
+        run: |
+          cd src/firecrest-streamer
+          python - << 'EOF'
+          import toml, json, requests, sys
+          data = toml.load("pyproject.toml")
+          project = data.get("project", {})
+          name = project.get("name")
+          version = project.get("version")
```

Comment:
If I understood correctly, if we need to change the version, we must do it manually in the toml file and then push (which will trigger the pipeline according to [line 7](https://github.com/eth-cscs/firecrest-v2/pull/140/files#diff-824c10ebff108b1ad84295187ed438b315f5fd656309ab015348ebdcc1ebdb70R7), correct?